exercises/Sql-2-R.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: exercise
+title: R-SQL 2
+subtitle: Automate Query
+language: R
+---
+
+This is a follow-up to [R-SQL 1]({{ site.baseurl }}/exercises/Sql-2-R).
+
+Dr. Undómiel agrees with you that the difference in male and female *D. spectabilis* hind foot length and weight seems pretty small, but wants to have a 
+reference point for comparison. She wants you to find the male and female hind 
+foot length and weight for all species of rodent. 
+
+1. Generate a list of all of the rodent `species_id` at Portal.
+2. Write a function that generalizes the queries you used to `SELECT` the 
```

Comment:
> So, Sql-2-R.md is ok, as is?

Yes, sorry for the confusion.

> Should I generalize the task descriptions here, too?

I think these are OK since they haven't seen most of the things you describe here, but you could do:
* `SELECT` -> select
* get rid of the reference to using data.frame() and just mention that rbind may be useful.